addons/base_import/models/base_import.py

```diff
@@ -223,11 +223,10 @@ def _read_xls_book(self, book):
                 elif cell.ctype is xlrd.XL_CELL_BOOLEAN:
                     values.append(u'True' if cell.value else u'False')
                 elif cell.ctype is xlrd.XL_CELL_ERROR:
+                    # Both the row_index and cell_index have a +1 for human readable input (computers start counting at 0, we start at 1).
                     raise ValueError(
-                        _("Error cell found while reading XLS/XLSX file: %s") %
-                        xlrd.error_text_from_code.get(
-                            cell.value, "unknown error code %s" % cell.value)
-                    )
+                        _("Error cell found while reading XLS/XLSX file. Failed on line %s, column %s. Error code: %s") %
```

Comment:
better `row` instead of `line`

Reply:
Since we're already changing the text, I would suggest something a bit less technical, e.g.
```
_("Invalid cell value at row %(row)s, column %(col)s: %(cell_value)s") % (...)
```

Reply:
Also I don't think you're supposed to directly use the meaningless error code (`42` in your example) but to convert it to a user representation (e.g. `'#N/A'`) via xlrd's `error_text_from_code` map, which will normally correspond to what the user will see on the spreadsheet:
```python
In [2]: xlrd.error_text_from_code
Out[2]: 
{0: '#NULL!',
 7: '#DIV/0!',
 15: '#VALUE!',
 23: '#REF!',
 29: '#NAME?',
 36: '#NUM!',
 42: '#N/A'}


```